CHANGELOG.md

```diff
@@ -43,6 +43,12 @@ and this project adheres to
   T2S template to set bit 27 of `MSR_IA32_ARCH_CAPABILITIES` (`RFDS_NO`) to 1
   since it assumes that the fleet only consists of processors that are not
   affected by RFDS.
+- [#4388](https://github.com/firecracker-microvm/firecracker/pull/4388): Avoid
```

Comment:
We can say that this saves us a VM exit at the beginning.

nit: vCPU is more common in our doc than VCPU.

Reply:
done